src/compas/datastructures/graph/graph.py

```diff
@@ -436,7 +436,9 @@ def add_node(self, key=None, attr_dict=None, **kwattr):
         Examples
         --------
         >>> graph = Graph()
-        >>> graph.add_node()
+        >>> key = graph.add_node()
```

Comment:
i would prefer not to use `key` anymore, since it is a reference to an implementation detail.
can it not be `>>> node = graph.add_node()`?

Reply:
for example, in potential subsequent functions, i find it makes a difference...
```python
x = graph.node_attribute(node, 'x')
```

Reply:
btw, would also like to refactor this in the names of function and method params...

Reply:
👍